test/core/vm/actor/builtin/v0/payment_channel/payment_channel_actor_test.cpp

```diff
@@ -155,7 +157,9 @@ TEST_F(PaymentChannelActorTest, ConstructSuccess) {
   EXPECT_EQ(state.to_send, 0);
   EXPECT_EQ(state.settling_at, 0);
   EXPECT_EQ(state.min_settling_height, 0);
-  EXPECT_EQ(state.lanes.size(), 0);
+
+  EXPECT_OUTCOME_TRUE(lanes_size, state.lanes.size());
+  EXPECT_EQ(lanes_size, 0);
```

Comment:
```suggestion
  EXPECT_OUTCOME_EQ(state.lanes.size(), 0);
```

Reply:
I couldn't. size() returns outcome::result<uint64_t> not just uint64_t. And test comparator can't compare these values.